assets/s/pasteclick.js

```diff
@@ -12,8 +12,14 @@ function getPastePre(){
 function highlightPre(pastePre){
 	hljs.highlightBlock(document.body);  	
 	hljs.highlightBlock(pastePre);
-	}
+}
 function initPaste(){
 	inPastePre = getPastePre();
 	highlightPre(inPastePre);
 }
+function codeType(){
+	if(window.location.hash){
+		var lang = window.location.hash.substr(1);
```

Comment:
Without a listener to call this, adding the `#language` won't change the language highlighting, look at how changestyle works: https://github.com/PacketFire/paste-click/blob/5d4dfa8adf318824783f75541195c262f5b4901a/assets/s/pasteclick.js#L2 to call the function when it changes.